frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationBody.tsx

```diff
@@ -17,6 +17,7 @@ const UtilizationAxis: React.FC<UtilizationAxisProps> = ({ timestamps = [] }) =>
     <div ref={containerRef}>
       {!!timestamps.length && (
         <ChartAxis
+          containerComponent={<ChartContainer title="time axis" />}
```

Comment:
Needs to be translated.

Reply:
If this is changing more than test code, we need to have QE review. A Bugzilla bug would be good since it has accessible fixes.